msteams-platform/concepts/tabs/tabs-overview.md

```diff
@@ -1,27 +1,33 @@
 ---
-title: Get started with tabs
-description: Describes how to get started with tabs in Microsoft Teams
-keywords: get started tabs teams
+title: Develop tabs
+description: Describes how to get started developing tabs in Microsoft Teams
+keywords: teams tabs development
 ---
 
-# Get started with tabs for Microsoft Teams
+# Develop tabs for Microsoft Teams
```

Comment:
Should it be 'for' or 'in'?

Reply:
I've had it both ways and finally settled on "for", for two reasons: A dev doesn't use Teams as a development environment (which "in" could be interpreted as meaning), and it matches the H1s of the peer topics.